### ISSUE TITLE
Investigate Electronite

### ISSUE THREAD

**Issue by tjcouch-sil** [paranext/paranext-core#9]
Let's investigate if Electronite is a good fit for us.

- Is it maintained well?
- Is having Graphite support useful and worth it?
- Can it fit into our ecosystem?
- Is there an electronite builder tool?

https://github.com/unfoldingWord/electronite

**Comment by irahopkinson** [comment]
`electronite` is a drop in replacement for `electron`. Specifically, it is a fork of Electron that has the chromium UI patched to support [Graphite](https://scripts.sil.org/cms/scripts/page.php?site_id=projects&item_id=graphite_about) Fonts.

- it is updated only when uW does a major release of translationCore (and they therefore use a later version of electron) and it is a manual process of a couple of days. E.g. for v22+, at the moment there are `electronite` releases for '22.0.0-graphite', '22.0.1-graphite',  '22.0.3-graphite' but `electron` has further releases of '22.0.2', '22.1.0', '22.2.0', '22.2.1', '22.3.0', '23.0.0', '23.1.0' that `electronite` doesn't yet support. Note they skipped '22.0.2'. However, they are open to help with this.
- you also must use `electronite-packager` with it, which is a fork of `electron-packager`. There is only one version of this '15.4.0-graphite' so far whereas `electron-packager` currently has these additinal versions '15.5.0', '15.5.1', '15.5.2', '16.0.0', '17.0.0', '17.1.0', '17.1.1'.

So we would need to change our build system to use `electron-packager` first, then `electronite` and `electronite-packager` could be dropped it.

**Comment by irahopkinson** [comment]
I asked for more info about the state of adding Graphite features into Open Type (OT) to the WSTech team in Slack. I record important comments here:
Chris noted - "The only scripts that I can think of off hand that still require Graphite for proper rendering are Nastaliq and Lanna"
Martin noted that progress is "going slowly but not completely stuck". But they don't have any idea on the timeframe. "Yes it is still the plan and our desire for Harfbuzz to fully integrate Graphite in a way that all can use it and Awami [font] can just be used wherever."
Bobby noted - "A Tamil font that I worked on has both OT and Graphite (GR). I thought of removing the GR, as the OT with HarfBuzz (HB) is sufficient. But Paratext seems to have an older version of HB, and the font did not work well there with just OT. So I kept the GR, and it works well in Paratext. So Tamil does not require GR, but Tamil in Paratext does. Keep in mind the integration of Graphite is not into OT per se, but into HarfBuzz, which is an OT engine. Other OT engines, such as DirectWrite from Microsoft, Lipka from Adobe, and Apple has one, are not getting the Graphite integration."
